src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
```

Comment:
BulldozerProducer isn't related to model, replace and check other producers

Reply:
not fixed